scipy/integrate/_quadpack_py.py

```diff
@@ -702,16 +781,27 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
 
     """
 
+    if epsabs != inspect.signature(dblquad).parameters.get('epsabs').default:
+        msg = "Passing `epsabs` explicitly will be deprecated in a future" \
+              "release. Please pass it via the `opts` argument. `epsabs`" \
+               "will be ignored."
+        warnings.warn(msg, DeprecationWarning, stacklevel=2)
```

Comment:
Here and below I'm not sure if ``stacklevel=2`` is the correct option.

Reply:
Please show the output. It should refer to the user's line of code in which `dblquad` was called.